website/addons/dataverse/tests/test_views.py

```diff
@@ -12,17 +12,18 @@
 from framework.auth.decorators import Auth
 
 from website.util import api_url_for
+from website.addons.base.testing import views
+from website.addons.dataverse.model import DataverseProvider
 from website.addons.dataverse.serializer import DataverseSerializer
 from website.addons.dataverse.tests.utils import (
     create_mock_connection, DataverseAddonTestCase, create_external_account,
 )
 from website.oauth.models import ExternalAccount
 
-
-class TestDataverseViewsAuth(DataverseAddonTestCase):
+class TestAuthViews(DataverseAddonTestCase):
 
     def test_deauthorize(self):
-        url = api_url_for('dataverse_remove_user_auth',
+        url = api_url_for('dataverse_import_auth',
```

Comment:
It's the same route, just a different method (`DELETE` rather than `PUT`). But I'll change it.